Makefile

```diff
@@ -15,15 +15,25 @@ gen-certs:
 generate:
 	go generate ./...
 
-.PHONY: build
+.PHONY: build-all build-attesters build
+build-all: build-attesters build
```

Comment:
Instead of having `build-attesters` and `build` targets, could we have `build-subattesters` and `build-leadattester`, or something like `build-sa` and `build-la`?